src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/product/create.yaml

```
@@ -191,6 +191,65 @@ sylius_twig_hooks:
                     auto_open: false
                 priority: 0
 
+        'sylius_admin.product.create.content.form.sections.attributes':
+            autocomplete:
+                template: '@SyliusAdmin/product/form/sections/attributes/autocomplete.html.twig'
+                priority: 100
+            list:
+                template: '@SyliusAdmin/product/form/sections/attributes/list.html.twig'
+                priority: 0
+
+        'sylius_admin.product.create.content.form.sections.attributes.autocomplete':
+            input:
+                template: '@SyliusAdmin/product/form/sections/attributes/autocomplete/input.html.twig'
+                priority: 100
+            button:
+                template: '@SyliusAdmin/product/form/sections/attributes/autocomplete/button.html.twig'
+                priority: 0
+
+        'sylius_admin.product.create.content.form.sections.attributes.list':
+            name:
+                template: '@SyliusAdmin/product/form/sections/attributes/list/name.html.twig'
+                priority: 100
+            translations:
+                template: '@SyliusAdmin/product/form/sections/attributes/list/translations.html.twig'
+                priority: 0
+
+        'sylius_admin.product.create.content.form.sections.attributes.list.name':
```

I'm not sure about it but maybe something like
```suggestion
        'sylius_admin.product.create.content.form.sections.attributes.list.manage':
```
or
```suggestion
        'sylius_admin.product.create.content.form.sections.attributes.list.switch':
```
will be more accurate 🤔